docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,138 +1,243 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
-1. <a name="9"></a>Starting with Visual Studio 17.2, a `foreach` using a ref struct enumerator type reports an error if the language version is set to 7.3 or earlier.
+## Foreach enumerator as a ref struct
 
-2. <a name="8"></a>Starting with Visual Studio 17.2, an async `foreach` prefers to bind using a pattern-based `DisposeAsync()` method rather than `IAsyncDisposable.DisposeAsync()`.
-    For instance, the `DisposeAsync()` will be picked, rather than the `IAsyncEnumerator<int>.DisposeAsync()` method on `AsyncEnumerator`:
-    ```csharp
-    await foreach (var i in new AsyncEnumerable())
-    {
-    }
+***Introduced in .NET SDK 6.0.300, Visual Studio 2022 version 17.2.***. A `foreach` using a ref struct enumerator type reports an error if the language version is set to 7.3 or earlier.
```

Comment:
Is it expected to have two periods after the VS version here and in several other places below?

Reply:
Nope. I'll fix that.